### PR TITLE
data: ShivanshPratap

### PR DESCRIPTION
**Pull request by ShivanshPratap**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

You all are doing a great job. I would like my LinkFree profile to be updated.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5171"></a>

### CODE REVIEW

**Review by sidharthviz — DISMISSED** [review]
Looks Good 🎉

File: data/ShivanshPratap.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+	"name": "Shivansh Pratap",
+	"displayStatsPublic": true,
```
Comment (imbeshat) [inline data/ShivanshPratap.json:3]:
```suggestion
```

**Review by imbeshat — CHANGES_REQUESTED** [review]
```displayStatsPublic``` is not required.

**Review by ShivanshPratap — COMMENTED** [review]
removed displayStatsPublic

**Review by SurajPratap10 — APPROVED** [review]
LGTM

**Pull request merged**